### DISCUSSION TITLE
LCD / Messages not working on ver 4.8.0

### DISCUSSION THREAD

**Discussion by AaronNoletPDIT** [theyosh/TerrariumPI#859] [Q&A] [answered]
Cannot get messages displayed on LCD... see screenshots below.

LCD Does display the Title with "Show Title" Option. But that is it... Nothing Else displays.

![image](https://github.com/theyosh/TerrariumPI/assets/50311191/76f68b2c-933d-4d25-bf5c-a39fcb5ec568)
![image](https://github.com/theyosh/TerrariumPI/assets/50311191/7744d962-c0d8-4a06-94fd-ca8c794d85e6)

**Comment by theyosh** [comment]
Hi, I need to look into this. This is not used that much, and the code is not the best :(

**Comment by theyosh** [comment, marked as answer]
Found the bug. Do a `git pull` and restart. It should work again.
